### PR TITLE
Formatting changes

### PR DESCRIPTION
**Pull request by soundarya98**
Minor formatting edits: Changed `if(X)` to `if X`

### CODE REVIEW

File: dice_ml/model.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def __init__(self, model=None, model_path='', backend=BackEndTypes.Tensorflow1,
             )
 
         self.model_type = model_type
-        if((model is None) & (model_path == '')):
+        if (model is None) and (model_path == ''):
```
Comment (gaugup) [inline dice_ml/model.py:43]: Probably you could drop these parenthesis as well. See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.

File: dice_ml/explainer_interfaces/dice_tensorflow2.py [hunk 2]
```diff
@@ -378,8 +378,8 @@ def stop_loop(self, itr, loss_diff):
         """Determines the stopping condition for gradient descent."""
 
         # intermediate projections
-        if((self.project_iter > 0) & (itr > 0)):
-            if((itr % self.project_iter) == 0):
+        if self.project_iter > 0 and itr > 0:
+            if (itr % self.project_iter) == 0:
```
Comment (gaugup) [inline dice_ml/explainer_interfaces/dice_tensorflow2.py:382]: Probably you could drop these parenthesis as well. See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.

File: dice_ml/explainer_interfaces/dice_tensorflow1.py [hunk 3]
```diff
@@ -477,8 +477,8 @@ def stop_loop(self, itr, loss_diff):
         """Determines the stopping condition for gradient descent."""
 
         # intermediate projections
-        if((self.project_iter > 0) & (itr > 0)):
-            if((itr % self.project_iter) == 0):
+        if self.project_iter > 0 and itr > 0:
+            if (itr % self.project_iter) == 0:
```
Comment (gaugup) [inline dice_ml/explainer_interfaces/dice_tensorflow1.py:481]: Probably you could drop these parenthesis as well. See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.

File: dice_ml/explainer_interfaces/dice_pytorch.py [hunk 4]
```diff
@@ -376,8 +376,8 @@ def stop_loop(self, itr, loss_diff):
         """Determines the stopping condition for gradient descent."""
 
         # intermediate projections
-        if((self.project_iter > 0) & (itr > 0)):
-            if((itr % self.project_iter) == 0):
+        if self.project_iter > 0 and itr > 0:
+            if (itr % self.project_iter) == 0:
```
Comment (gaugup) [inline dice_ml/explainer_interfaces/dice_pytorch.py:380]: Probably you could drop these parenthesis as well. See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.

File: dice_ml/data_interfaces/public_data_interface.py [hunk 5]
```diff
@@ -404,10 +404,10 @@ def get_decimal_precisions(self, output_type="list"):
...
         for ix, col in enumerate(self.continuous_feature_names):
-            if ((self.continuous_features_precision is not None) and (col in self.continuous_features_precision)):
+            if (self.continuous_features_precision is not None) and (col in self.continuous_features_precision):
                 precisions[ix] = self.continuous_features_precision[col]
                 precisions_dict[col] = self.continuous_features_precision[col]
-            elif ((self.data_df[col].dtype == np.float32) or (self.data_df[col].dtype == np.float64)):
+            elif (self.data_df[col].dtype == np.float32) or (self.data_df[col].dtype == np.float64):
                 modes = self.data_df[col].mode()
                 maxp = len(str(modes[0]).split('.')[1])  # maxp stores the maximum precision of the modes
                 for mx in range(len(modes)):
```
Comment (gaugup) [inline dice_ml/data_interfaces/public_data_interface.py:413]: Probably you could drop these parenthesis as well. See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.

**Review by gaugup — CHANGES_REQUESTED** [review]
See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.

**Review by amit-sharma — APPROVED** [review]

**Pull request merged**